design-library/src/components/BccGraphic/BccGraphicPoster.vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+defineProps<{
+  bannerSrc?: string;
+  logoSrc?: string;
+  rounding: "md" | "xl";
+}>();
+</script>
+
+<template>
+  <div class="relative h-full">
+    <div
+      class="bcc-graphic-poster"
+      :style="[
+        bannerSrc ? `background-image: url(${bannerSrc});` : '',
+        rounding === 'md' ? 'border-radius: 6px;' : rounding === 'xl' ? 'border-radius: 12px;' : '',
+      ]"
```

Comment:
The `rounding` options should be classes as well I think since they're pretty important to the layout of the poster. Consider maybe adding the default `xl` rounding to the base `bcc-graphic-poster` class and only add a specific class for the non-default option.

```suggestion
      :style="[ bannerSrc ? `background-image: url(${bannerSrc});` : '',]"
      :class="{ 'bcc-graphic-poster-rounding-md': rounding == 'md' }"
```

Reply:
This solution seems like a lot of extra classes for a more complicated solution.

Reply:
I would write the component in the same way if it's local to a project (maybe use Tailwind `rounded` classes instead), but for this library it's slightly different as we need to accommodate different use cases. Someone rendering this component out with CSS classes would always have to add the `border-radius` themselves now. It's not a huge issue in this case, but I think it's important to be consistent.

I understand if it feels like unnecessary nagging 😅 I'll push a small update to this branch myself, see the tests are also failing.